src/gfx/context.cpp

```diff
@@ -176,7 +181,7 @@ struct ContextMainOutput {
     swapchainDesc.format = swapchainFormat;
     swapchainDesc.width = newSize.x;
     swapchainDesc.height = newSize.y;
-    swapchainDesc.presentMode = WGPUPresentMode_Fifo;
+    swapchainDesc.presentMode = WGPUPresentMode_Immediate;
```

Comment:
isn't Fifo better?

Reply:
I think it still switches to that automatically on some platforms that enforce it. but we want to control our own frame-rate so we can target 120 or 240 for example for better UI response times